docs/running-in-ci.md

```diff
@@ -0,0 +1,239 @@
+# Running in CI
+
+[TOC]
+
+The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a free [Semgrep Community](https://semgrep.dev/manage) or paid Semgrep Team account. `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
```

Comment:
@pabloest @ievans For this section I think we could reduce the strength of our language to highlight that Semgrep works in CI (and has an epic wrapper with XYZ benefits), and optionally can be used with Semgrep App for notifications, org-wide management, etc. Thoughts?

Reply:
@dlukeomalley Do you mean to account for more use cases based on "manual CI" running instead of Semgrep App -based running? Wouldn’t users miss out on policy management, notification config, analytics, etc.? (which all require running in CI and an account)